ONSdigital/eq-survey-runner
# Template changes for MCI

**_What**_

Updates the mci schema and cover-page template to include the return-by date in the legal paragraph

**_How to test**_

1) Checkout the branch
2) View the landing page
3) Verify that the return date in the paragraph matches that at the top of the page

**_Who can test**_

Anyone except @weapdiv-david
